hibernate/hibernate-offline-startup/src/test/java/io/quarkus/ts/hibernate/startup/offline/test/OpenTelemetryHibernateOfflineStartupIT.java

```diff
@@ -0,0 +1,62 @@
+package io.quarkus.ts.hibernate.startup.offline.test;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.jupiter.api.Test;
+
+import io.quarkus.test.bootstrap.JaegerService;
+import io.quarkus.test.bootstrap.PostgresqlService;
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.QuarkusScenario;
+import io.quarkus.test.services.Container;
+import io.quarkus.test.services.Dependency;
+import io.quarkus.test.services.JaegerContainer;
+import io.quarkus.test.services.QuarkusApplication;
+import io.quarkus.test.utils.AwaitilityUtils;
+import io.restassured.response.Response;
+
+@QuarkusScenario
+public class OpenTelemetryHibernateOfflineStartupIT extends AbstractHibernateOfflineStartupIT {
+
+    @JaegerContainer(expectedLog = "\"Health Check state change\",\"status\":\"ready\"")
+    static final JaegerService jaeger = new JaegerService();
+
+    @Container(image = "${postgresql.latest.image}", port = 5432, expectedLog = "listening on IPv4 address", builder = FixedPortResourceBuilder.class)
+    static final PostgresqlService db = new PostgresqlService().setAutoStart(false);
+
+    @QuarkusApplication(dependencies = {
+            @Dependency(artifactId = "quarkus-jdbc-postgresql"),
+            @Dependency(artifactId = "quarkus-opentelemetry")
+    })
+    static final RestService app = new RestService()
+            .withProperty("quarkus.application.name", "app")
+            .withProperty("quarkus.datasource.app_scope_credentials.jdbc.telemetry", "true")
+            .withProperty("quarkus.datasource.req_scope_credentials.jdbc.telemetry", "true")
+            .withProperty("quarkus.otel.exporter.otlp.traces.endpoint", jaeger::getCollectorUrl)
+            .withProperty("jdbc-url", "jdbc:postgresql://localhost:5432/mydb");
+
+    @Test
+    void testJdbcTraces() {
+        AwaitilityUtils.untilAsserted(() -> {
+            var response = thenRetrieveTraces();
+            var operationNames = response.jsonPath().getList("data.spans.operationName.flatten()", String.class);
+            assertThat(operationNames).contains("SELECT mydb.Article").contains("INSERT mydb.Article");
```

Comment:
I'm confused, DB is not started and you are checking traces for insert?

Or you are just expecting to run this test after tests from `AbstractHibernateOfflineStartupIT`?

Reply:
> I'm confused, DB is not started and you are checking traces for insert?
> 
> Or you are just expecting to run this test after tests from `AbstractHibernateOfflineStartupIT`?

It is annoying how you mention everything that is wrong :-D yes, this test is not independent and it requires `AbstractHibernateOfflineStartupIT` to run first (which happens by the default order), but there is no explicitly declared order. I made my life easier as it seems that parent methods run first and they already require scheme etc. I can produce some traces in this method instead if you want it....

Reply:
I think that `@TestMethodOrder(MethodOrderer.OrderAnnotation.class)` could be tricked to add the order only to this child method so that it always run last, I'll try it

Reply:
yep, works